Poll.py

```diff
@@ -0,0 +1,48 @@
+import discord
+from settings import *
+from checks import *
+
+
+class Poll(commands.Cog):
```

Comment:
Add a command to add poll channel if has permission to

Reply:
See #4